test/e2e/rule_test.go

```diff
@@ -62,9 +66,22 @@ groups:
 			"replica":    "2",
 		},
 	}
+	expAlertLabels := []model.LabelSet{
+		{
+			"severity":  "page",
+			"alertname": "AlwaysFiring",
+			"replica":   "1",
+		},
+		{
+			"severity":  "page",
+			"alertname": "AlwaysFiring",
+			"replica":   "2",
+		},
+	}
```

Comment:
In the future we'll want to support a `--replica-label` flag for the rule node as well, which just strips the replica label.

Reply:
:+1: